addon/classes/Column.js

```diff
@@ -129,11 +130,37 @@ export default class Column extends EmberObject.extend({
 
   /**
    * The minimum width (in px) that this column can be resized to.
-   * @property minResizeWidth
+   * @property _minResizeWidth
    * @type {Number}
    * @default 0
+   * @private
+   */
+  _minResizeWidth: 0,
+
+  /**
+   * The minimum width that this column can be resized to.
+   * @property minResizeWidth
+   * @type {String}
+   * @default '0px'
    */
-  minResizeWidth: 0,
+  minResizeWidth: computed({
+    get() {
+      return this.get('_minResizeWidth');
+    },
+    set(key, value) {
+      if (typeOf(value) === 'string') {
+        let [, quantity, units] = value.match(/(\d+)(px)?/) || [];
+        if (units !== 'px') {
+          warn('`value` attribute is interpreted as px regardless of provided units', {
+            id: 'ember-light-table.classes.Column'
+          });
```

Comment:
```js
const [, quantity, units] = value.match(/^\s*(\d+)([a-z]+|%)?\s*$/) || [];
assert(
  `'minResizeWidth' can only interpret px unit values. You passed '${quantity}${unit}'.`,
  isEmpty(units) || units === 'px'
);
```

Being more restrictive here can help prevent subtle bugs. We should throw an assertion here, since `1rem` would be interpreted as `1px` in the original code, which does not make sense.